bugbear.py

```diff
@@ -375,6 +379,26 @@ def check_for_b902(self, node):
                 )
             )
 
+    def check_for_b903(self, node):
+        if (len(node.body) != 1 or
+                not isinstance(node.body[0], ast.FunctionDef) or
+                node.body[0].name != '__init__'):
```

Comment:
Ah, the eternal `if` over-indent discussion! I'll comply with your preference and I'll add support for `__doc__` too.